server/app.py

```diff
@@ -40,5 +49,7 @@ def create_app():
     api.add_resource(UserResource, '/users')
     api.add_resource(LoginResource, '/users/login')
     api.add_resource(MealItemResource, '/meal_items')
+    api.add_resource(LogoutResource, '/users/logout')
+    api.add_resource(TokenRefreshResource, '/token/refresh')
```

Comment:
I feel like /token/refresh should be a constant variable at the top of file since it's used twice